pull_request_template.md

```diff
@@ -2,15 +2,15 @@
 
 > Please provide all of the details below. Pull requests missing any of the information below will be closed!
 
-**Sumarise what this PR fixes:** < enter details >
+**Sumarise what this PR fixes:** Updated "Contact Us" Heading above Contact Form. Updated Conflicts.
 
-**Fixes #:** < Enter Issue ID/URL >
+**Fixes #:** [199](https://github.com/zero-to-mastery/Keiko-Corp/issues/199#issue-713611402)
```

Comment:
don't add PR information in this file. this is a template for PR. content of this file would be shown in pull request body when you open a new PR

Reply:
Got it.